pageserver/src/http/openapi_spec.yml

```diff
@@ -52,6 +52,29 @@ paths:
               schema:
                 type: object
 
+  /v1/deletion_queue/flush:
+    parameters:
+      - name: execute
+        in: query
+        required: false
+        schema:
+          type: boolean
+        description:
+          If true, attempt to execute deletions.  If false, just flush deletions to persistent deletion lists.
+    put:
+      description: Execute any deletions currently enqueued
+      security: []
+      responses:
+        "200":
+          description: |
+            Flush completed: if execute was true, then enqueued deletions have been completed.  If execute was false,
+            then enqueued deletions have been persisted to deletion lists, and may have been completed.
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
Hmm, I was thinking to leave this in the external API in case we wanted it for some operational reason, like if we were going to hard-kill a pageserver but wanted to flush deletions first.  I could be persuaded either way though